packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2681,26 +2681,30 @@ String _concatStrings({
 ///    [SemanticsSortOrder] manages.
 ///  * [OrdinalSortKey] for a sort key that sorts using an ordinal.
 class SemanticsSortOrder extends Diagnosticable implements Comparable<SemanticsSortOrder> {
+  /// Creates an object that describes the sort order for a [Semantics] widget.
+  ///
   /// Only one of `key` or `keys` may be specified, but at least one must
   /// be specified. Specifying `key` is a shorthand for specifying
-  /// `keys = <SemanticsSortKey>[key]`.
+  /// `keys: <SemanticsSortKey>[key]`.
   ///
-  /// If [discardParentOrder] is set to true, then the
-  /// [SemanticsSortOrder.keys] will replace the list of keys from the parents
-  /// when merged, instead of extending them.
+  /// If [discardParentOrder] is set to true, then the [SemanticsSortOrder.keys]
+  /// will _replace_ the list of keys from the parents when merged, instead of
+  /// extending them.
   SemanticsSortOrder({
     SemanticsSortKey key,
     List<SemanticsSortKey> keys,
     this.discardParentOrder = false,
-  })
-    : assert(key != null || keys != null, 'One of key or keys must be specified.'),
```

Comment:
Damn dartfmt.  I really wish they would fix that one thing: pretty much everything else is working.

Reply:
file a bug on dartfmt, see what bob says :-)